src/plugins/intel_gpu/tests/functional/behavior/model_cache.cpp

```diff
@@ -304,4 +307,32 @@ INSTANTIATE_TEST_SUITE_P(smoke_CheckWeightlessCacheAccuracyLowPrecision,
                                             ::testing::ValuesIn(low_precision_dtypes)),
                          CheckWeightlessCacheAccuracy::get_test_case_name);
 
+TEST(smoke_CheckWeightlessCacheAccuracy, ConstantFoldingAttrPropagation) {
+    std::vector<uint64_t> data(4, 0);
```

Comment:
should it be `std::vector<ov::float16> data(4, 0.f);`?

Reply:
Fixed.